website/src/css/customTheme.scss

```
@@ -182,6 +182,19 @@ hr {
       top: -1px;
       position: relative;
     }
+
+    .img-inline {
+      width: 300px;
+      float: right;
+      margin: -30px 4px 0;
+    }
+
+    @media screen and (max-width: 500px) {
```

```suggestion
    @media screen and (max-width: 960px) {
```

`960px` is a breakpoint used on website for the mobile layout switch.